src/main/kotlin/com/itiviti/DotnetPlugin.kt

```diff
@@ -38,7 +38,7 @@ class DotnetPlugin: Plugin<Project> {
             }.exitValue
         }
 
-        private fun getMajorVersion(version: String) = version.substringBeforeLast('.').toDouble()
+        private fun getMajorVersion(version: String) = version.substringBeforeLast('-').substringBeforeLast('.').toDouble()
```

Comment:
`.substringBeforeLast('-')` will return empty, it should be 
`.substringBeforeLast('-', version)`

Reply:
Indeed....